reducer/src/main/java/com/graphicsfuzz/reducer/reductionopportunities/VariableDeclToExprReductionOpportunities.java

```diff
@@ -53,4 +55,28 @@ public VariableDeclToExprReductionOpportunities(TranslationUnit tu, ReducerConte
     finder.visit(tu);
     return finder.getOpportunities();
   }
+
```

Comment:
Could you add a comment to the top of this class briefly explaining, with a couple of examples, the kinds of opportunity that this pass will look for?

Reply:
Updated.